week-2/F-logical-operators/exercise.js

```diff
@@ -8,11 +8,15 @@
 // Do not change these two statement
 var htmlLevel = 8;
 var cssLevel = 4;
+function  satisfiesRequirements(htmlLevel) {
+  if (htmlLevel>4 && htmlLevel<=8){
+    return "true";
+  }
+}
 
-// Finish the statement to check whether HTML, CSS knowledge are above 5
-// (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = 8;
+var cssLevelAbove5 = 4;
+function satisfiesRequirements(cssAndHtmlAbove5) 
```

Comment:
What happens when you run this? You've defined `satisfiesRequirements` twice but it isn't used. I don't think you need to write a function here. Just set the values (check the "hint" in the comment). They should be boolean values (true or false), rather than numbers.